### DISCUSSION TITLE
Deployment problem

### DISCUSSION THREAD

**Discussion by CreatorYo** [vercel/community#7389] [Help]
### Summary

I am encountering a problem whenever I deploy, the page does not load and I have vercel.json and that still doesn't seem to work. I have spent approx 2 hours on resolving this and still can't.

### Example

https://api.creatoryogames.com

### Steps to Reproduce

**Vercel.json **

{
  "version": 2,
  "builds": [
    { 
      "src": "/server.mjs", 
      "use": "@vercel/node" 
    }
  ],
  "routes": [
    { "handle": "filesystem" },
    { "src": "/(.*)", "dest": "/server.mjs" }
  ]
}

---------

Error message: ❗️  Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings

**Comment by pawlean** [comment]
Hi @CreatorYo,

According to the error message that you shared:

> Error message: ❗️ Due to builds existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings

It looks like it may be due to different configurations in different places e.g. Project settings and the `vercel.json`? Looks like they differ from each other. It's worth removing one and trying to see if your app now works?

**Comment by CreatorYo** [comment]
Already tried that and didn't work.

[a comment by an automated account is not shown]
